source/extensions/filters/http/basic_auth/basic_auth_filter.cc

```diff
@@ -85,6 +84,16 @@ Http::FilterHeadersStatus BasicAuthFilter::decodeHeaders(Http::RequestHeaderMap&
   return Http::FilterHeadersStatus::Continue;
 }
 
+bool BasicAuthFilter::validateUser(const UserMap* users, absl::string_view username,
```

Comment:
Just keep using `const UserMap& users` in this method because this should never be null.

/wait

Reply:
Ok, done